src/core/document.js

```diff
@@ -1815,6 +1822,15 @@ class PDFDocument {
       }
     }
 
+    if (
+      parentRef &&
+      isName(field.get("Subtype"), "Widget") &&
+      !field.has("Parent")
```

Comment:
Nit: Maybe swap these lines around, to do the "cheaper" check first?
```suggestion
      !field.has("Parent") &&
      isName(field.get("Subtype"), "Widget")
```